src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -768,7 +768,8 @@ TTL7411 = 7411 : triple 3 entrées ET porte d'entrée
 TTL74125 = 74125 : tampon quadruple bus, sorties à trois états, validation négative
 TTL74157 = 74157 : sélecteur de données quadruple de 2 lignes à 1 ligne
 TTL74158 = 74158 : quadruple 2-lignes to 1 ligne sélecteur, Sorties inversées
```

Comment:
"quadruple 2-lignes to 1 ligne sélecteur, Sorties inversées" -> "sélecteur quadruple de 2 lignes à 1 ligne, sorties inversées"

Reply:
ma grammaire est pas bon, mais j'ai essayé